dotty-tensorflow/src/main/scala/me.shadaj.scalapy.tensorflow/example/GradientDescentOptimizerExample.scala

```diff
@@ -1,17 +1,19 @@
 package me.shadaj.scalapy.tensorflow.example
 
 import me.shadaj.scalapy.py
-import me.shadaj.scalapy.tensorflow.scala.utils.Modules._
-import me.shadaj.scalapy.tensorflow.{Tensor, nd2Tensor}
-import me.shadaj.scalapy.tensorflow.scala.utils.ContextManager
-
+import me.shadaj.scalapy.tensorflow.scala.utils.Modules
+import me.shadaj.scalapy.tensorflow.api.Tensor
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.CloseableResourceManager
+import me.shadaj.scalapy.tensorflow.api.Tensor.{TensorToPyTensor, nd2Tensor}
+import me.shadaj.scalapy.tensorflow.{nd2Tensor => nd2TensorPy}
+import me.shadaj.scalapy.tensorflow.api.TensorFlow
 import scala.language.implicitConversions
 
 object GradientDescentOptimizerExample extends Runnable {
 
   def run(): Unit = {
-    val tf = tensorflow
-    val np = numpy
+    val tf = new TensorFlow()
```

Comment:
`np` and `loss` are defined but used only once, maybe you could inline them?

Reply:
no, I don't want to touch `loss` because it nicely commented but i can do sth with `np`